### ISSUE TITLE
a11y: Add Missing alt on vehicle marker images

### ISSUE THREAD

**Issue by tarunsinghofficial** [OneBusAway/wayfinder#483]
The <img> tags inside Leaflet vehicle marker icons have no alt attribute and this fails WCAG 1.1.1.
